tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -25,21 +35,23 @@ public void run() {
 
     @Override
     public void process(final Socket connection) {
-        try (final var inputStream = connection.getInputStream();
-             final var outputStream = connection.getOutputStream()) {
+        try (final InputStream inputStream = connection.getInputStream();
+             final OutputStream outputStream = connection.getOutputStream();
+             final BufferedReader inputBufferedReader = new BufferedReader(new InputStreamReader(inputStream));) {
 
```

Comment:
```suggestion
             final BufferedReader inputBufferedReader = new BufferedReader(new InputStreamReader(inputStream))) {
```
;이 없어도 될 거 같아요! 그리고 의미 없는 공백이 있는 것 같네요